jsons/1.12.2.json

```diff
@@ -719,6 +719,6 @@
     "minecraftArguments": "--username ${auth_player_name} --version ${version_name} --gameDir ${game_directory} --assetsDir ${assets_root} --assetIndex ${assets_index_name} --uuid ${auth_uuid} --accessToken ${auth_access_token} --userType ${user_type} --versionType ${version_type}",
     "minimumLauncherVersion": 18,
     "releaseTime": "2017-09-18T08:39:46+00:00",
-    "time": "2018-02-15T16:26:45+00:00",
+    "time": "2018-04-02T07:39:26+00:00",
```

Comment:
This entire file change is unrelated to your pull request and should be excluded

Reply:
That was automatically done by gradlew genPatches

Reply:
That is not a valid reason for it to be included in your pull request..

Reply:
But the build time, size, and checksum _were_ changed as a result of this PR, so it should change, right?